src/Person.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import PersonName from './PersonName';
+
+const Person = ({ person, isSelected, selectPerson }) => {
+  const trClassName = (currentPerson) => {
+    let name = 'person';
+
+    name += ` person--lived-in-${currentPerson.century}`;
+    name += isSelected ? ' person--selected' : '';
+
+    return name;
+  };
+
+  const tdNameClassName = (currentPerson) => {
+    let name = '';
+
+    if (currentPerson.born < 1650) {
+      name += 'person--born-before-1650';
+    }
+
+    if (currentPerson.died > 1800) {
+      name += 'person--died-after-1800';
+    }
+
+    name += currentPerson.sex === 'm' ? ' person--man' : ' person--woman';
+
+    return name;
+  };
+
+  return (
+    <tr
+      className={trClassName(person)}
+      onClick={selectPerson}
+    >
+      <td>{person.id}</td>
+      <PersonName
+        person={person}
+        tdNameClassName={tdNameClassName}
+      />
+      <td>{person.sex}</td>
+      <td>{person.born}</td>
+      <td>{person.died < Infinity ? person.died : ''}</td>
+      <td
+        className={person.age >= 65 ? 'person--old' : ''}
```

Comment:
ось так це б виглядало з лібою classnames
```suggestion
        className={cn({ 'person-old': person.age >= 65 })} 
```